src/Components/NavBar/NavBar.tsx

```diff
@@ -28,18 +32,31 @@ import {
   SoloIcon,
   Spacer,
 } from "@artsy/palette"
+import { TrackingProp } from "react-tracking"
 
 interface NavBarProps {
-  user?: User
+  tracking?: TrackingProp
 }
 
-export const NavBar: React.FC<NavBarProps> = () => {
+export const NavbarContext = React.createContext<{
+  tracking?: TrackingProp
+  Schema?: typeof Schema
+}>({})
+
+export const NavBar = track()(({ tracking }: NavBarProps) => {
   const { mediator, user } = useContext(SystemContext)
   const [showMobileMenu, toggleMobileNav] = useState(false)
   const isLoggedIn = Boolean(user)
 
+  const navBarContextValue = useMemo(() => {
+    return {
+      tracking,
+      Schema,
+    }
+  }, [])
```

Comment:
Was this actually needed to avoid re-renders? Is it the `tracking` value that changes?

Reply:
Since nothing ever changes `[]` will return the same object over and over.